src/Model/Post.php

```diff
@@ -595,6 +595,12 @@ protected function init() {
 
 					return false;
 				},
+				'hasPassword'               => function () {
+					return ! empty( $this->data->post_password );
+				},
+				'password'                  => function () {
+					return ! empty( $this->data->post_password ) ? $this->data->post_password : null;
+				},
```

Comment:
why did the capability check get removed from this field? Is it already protected by other logic in the model? Want to make sure we're not exposing passwords on posts to folks that shouldn't be able to access the password 🤔

Reply:
Yup, it's protected top-level by [`Model::has_restricted_cap()`](https://github.com/wp-graphql/wp-graphql/blob/3b326a5887c90e7e697618046a144a4701decaef/src/Model/Post.php#L252) and tested [here](https://github.com/wp-graphql/wp-graphql/blob/3b326a5887c90e7e697618046a144a4701decaef/tests/wpunit/PostObjectQueriesTest.php#L2453).